src/haz3lweb/view/Code.re

```diff
@@ -35,18 +35,18 @@ let of_grout = [Node.text(Unicode.nbsp)];
 let of_whitespace =
   Core.Memo.general(
     ~cache_size_bound=1000000, ((whitespace_icons, indent, content)) =>
-    if (content == Whitespace.linebreak) {
+    if (Whitespace.get_string(content) == Whitespace.linebreak) {
```

Comment:
need to use String.equal not == for string comparisons (didn't look thoroughly elsewhere so may not just be here)